README.md

```diff
@@ -13,7 +13,7 @@ All content must be freely licensed.
 
 ### Media
 
-* `Santa` CC BY-SA 3.0 by jordan4ibanez
+* `Skins`
```

Comment:
`skins`, not `Skins` ;)

Or, just remove the backticks. Your choice.

Reply:
That would be inconsistent with the others.

Reply:
I think this PR should be merged as-is and a separate one be made to fix _all_ of the media credits.

Reply:
Hmm... it seems like all the names under `Media` are inconsistent. In CTF documentation, backticks are used for mod names or other pieces of actual code (see the `Code` section for an example).

Reply:
> I think this PR should be merged as-is and a separate one be made to fix all of the media credits.

Agreed

Reply:
Up to you 🤷‍♀️